lib/tasks/populate_task.rake

```diff
@@ -71,8 +71,11 @@ namespace :hubstats do
       repo = repo_checker(args[:repo])
       puts "Adding contributors to " + repo.full_name
 
-      Hubstats::GithubAPI.client({:auto_paginate => true}).contribs(repo.full_name).each do |contributor|
-        cont = Hubstats::User.create_or_update(contributor)
+      users = Hubstats::GithubAPI.client({:auto_paginate => true}).contribs(repo.full_name)
+      unless users == "" # there are no contributors because there are no commits yet
+        users.each do |contributor|
+          cont = Hubstats::User.create_or_update(contributor)
+        end
```

Comment:
I think you can write this with one less block like 
```
users.each do |contributor|
  ...
end unless users == ""
```

Also is users a string or an enumerable?  If the latter, it would be more appropriate to call empty?

Reply:
I'm currently getting this:
```
NoMethodError: undefined method `each' for "":String
```

So I'm assuming it is a string.